### PR TITLE
Update navigation

### PR DESCRIPTION
**Pull request by ravingupta**
Rearranging of pages and On-Boarding process is done.

### CODE REVIEW

File: app/views/layout.erb [hunk 1]
```diff
@@ -318,7 +281,7 @@
                 </li>
                 <li><a href="#"><i class="pg-outdent"></i> Feedback</a>
                 </li>
-                <li><a href="#"><i class="pg-signals"></i> Help</a>
+                <li><a href="javascript:void(0);" onclick="javascript:startIntro();"><i class="pg-signals"></i> Help</a>
```
Comment (pranny) [inline app/views/layout.erb]: Should use a jQuery event handler here.

File: app/controllers/home_controller.rb [hunk 2]
```diff
@@ -2,77 +2,10 @@
...
-      enterprise_id: enterprise.id,
-      type: Octo::Counter::TYPE_MINUTE,
-      ts: 5.minutes.ago.utc..Time.now.floor.utc
-    ).group_by { |x| x.ts }
-    apihits.values.each_with_index do |counters, index|
-      @api_data << { x: index, y: counters.inject(0) { |sum, counter| sum += counter.count } }
-    end
-    if @api_data.count > 0
-      @api_count = @api_data[@api_data.length-1][:y]
+    if params['intro'] == 'true'
```
Comment (pranny) [inline app/controllers/home_controller.rb]: Reduce it to `@intro = params['intro'] == 'true'`

File: app/views/layout.erb [hunk 3]
```diff
@@ -281,7 +281,7 @@
                 </li>
                 <li><a href="#"><i class="pg-outdent"></i> Feedback</a>
                 </li>
-                <li><a href="javascript:void(0);" onclick="javascript:startIntro();"><i class="pg-signals"></i> Help</a>
+                <li><a href="javascript:void(0);" id="help"><i class="pg-signals"></i> Help</a>
```
Comment (pranny) [inline app/views/layout.erb]: Why do we need a javascript void here? Replace it with a `#`

File: app/views/layout.erb [hunk 4]
```diff
@@ -404,18 +404,28 @@
 
     <script type="text/javascript">
 
-      var $target = $('[data-pages="sidebar"]');
+      $(function() {
```
Comment (pranny) [inline app/views/layout.erb:407]: Wrap it inside `$(document).load`

Comment (ravingupta) [inline app/views/layout.erb:407]: '$(function()' and '$(document).load'
both have same meaning

**Pull request merged**